bin/cli.js

```diff
@@ -186,7 +186,8 @@ function invoke(env) {
   commander.parse(process.argv);
 
   Promise.resolve(pending).then(function() {
-    commander.help();
+    commander.outputHelp();
+    exit('');
```

Comment:
Maybe some more specific error message about command not found could be given.

Reply:
Done. Also realized then that chalk and console.error were called in the wrong order…

Also wasn't sure what your policy is regarding rebasing/amending. 
Sorry if I got that wrong.

Reply:
Eventually PR goes in as a squash merge so rebase/amend/etc. doesn't really matter anything goes.